{"uid":"363a0ad5985fb3bb","name":"test_default_sign_up_website_qr_type[-2048x2048-menu_link_qr_create]","fullName":"tests.tests_default_flow.test_default_sign_up_png_download_welcome_modal.TestDefaultSignUpFlowPng#test_default_sign_up_website_qr_type","historyId":"26928acc6112928343c01b7036cc4726","time":{"start":1735196885279,"stop":1735196885279,"duration":0},"status":"broken","statusMessage":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","statusTrace":"request = <SubRequest 'sign_up_fixture' for <Function test_default_sign_up_website_qr_type[-2048x2048-menu_link_qr_create]>>\nfake_email = 'wtl-automation5610258975261@test.com'\n\n    @pytest.fixture(scope=\"function\", params=os.getenv(\"LANGUAGES\", \"\").split(\",\"))\n    @allure.title(\"Sign up\")\n    def sign_up_fixture(request, fake_email):\n        language = request.param\n        base_url = f\"{ENV_URL}{language}/\"\n        email = fake_email\n>       request.instance.main_page.open_page(f\"{base_url}register/\")\n\nconftest.py:99: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nbase/base_page.py:10: in open_page\n    self.page.goto(url)\n/usr/local/lib/python3.12/dist-packages/playwright/sync_api/_generated.py:9002: in goto\n    self._sync(\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_page.py:553: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_frame.py:145: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x7f2de093f9e0>\ncb = <function Channel.send.<locals>.<lambda> at 0x7f2ddf8bbec0>\nis_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nE           Call log:\nE           navigating to \"None/register/\", waiting until \"load\"\n\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py:520: Error","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":21,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1735196871910,"stop":1735196872010,"duration":100},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"delete_output_dir","time":{"start":1735196872010,"stop":1735196872010,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"pytestconfig","time":{"start":1735196872010,"stop":1735196872010,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1735196872011,"stop":1735196872011,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Set up browser: chromium","time":{"start":1735196872011,"stop":1735196873048,"duration":1037},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1735196872011,"stop":1735196872011,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Delete user after test","time":{"start":1735196885283,"stop":1735196885283,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"fake_email","time":{"start":1735196885283,"stop":1735196885283,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Failed test artifacts","time":{"start":1735196885283,"stop":1735196885283,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"context","time":{"start":1735196885283,"stop":1735196885307,"duration":24},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"page","time":{"start":1735196885307,"stop":1735196885396,"duration":89},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Sign up","time":{"start":1735196885396,"stop":1735196885522,"duration":126},"status":"broken","statusMessage":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"\n\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/app/workspace/conftest.py\", line 99, in sign_up_fixture\n    request.instance.main_page.open_page(f\"{base_url}register/\")\n  File \"/app/workspace/base/base_page.py\", line 10, in open_page\n    self.page.goto(url)\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/sync_api/_generated.py\", line 9002, in goto\n    self._sync(\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_page.py\", line 553, in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_frame.py\", line 145, in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py\", line 59, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py\", line 520, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"setup","time":{"start":1735196885396,"stop":1735196885396,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"page::0","time":{"start":1735196885786,"stop":1735196885791,"duration":5},"status":"broken","statusMessage":"AttributeError: 'Function' object has no attribute 'rep_call'\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/app/workspace/conftest.py\", line 52, in page\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"context::0","time":{"start":1735196885791,"stop":1735196885892,"duration":101},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Delete user after test::0","time":{"start":1735196885892,"stop":1735196885893,"duration":1},"status":"broken","statusMessage":"requests.exceptions.MissingSchema: Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/app/workspace/conftest.py\", line 127, in delete_user_after_test\n    response = requests.post(DELETE_USER_URL, headers=headers, json=data)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/requests/api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/requests/sessions.py\", line 575, in request\n    prep = self.prepare_request(req)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/requests/sessions.py\", line 484, in prepare_request\n    p.prepare(\n  File \"/usr/local/lib/python3.12/dist-packages/requests/models.py\", line 367, in prepare\n    self.prepare_url(url, params)\n  File \"/usr/local/lib/python3.12/dist-packages/requests/models.py\", line 438, in prepare_url\n    raise MissingSchema(\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Failed test artifacts::0","time":{"start":1735196885894,"stop":1735196885896,"duration":2},"status":"broken","statusMessage":"AttributeError: 'Function' object has no attribute 'rep_call'\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/app/workspace/conftest.py\", line 72, in artifacts\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Set up browser: chromium::0","time":{"start":1735196938808,"stop":1735196938901,"duration":93},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"Default Sign Up Flow - PNG Format - chromium"},{"name":"tag","value":"smoke"},{"name":"parentSuite","value":"tests.tests_default_flow"},{"name":"suite","value":"test_default_sign_up_png_download_welcome_modal"},{"name":"subSuite","value":"TestDefaultSignUpFlowPng"},{"name":"host","value":"bd3659fb8bce"},{"name":"thread","value":"34-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.tests_default_flow.test_default_sign_up_png_download_welcome_modal"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"qr_create_method","value":"'menu_link_qr_create'"},{"name":"resolution","value":"'2048x2048'"},{"name":"sign_up_fixture","value":"''"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"42b79f2274506809","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196847782,"stop":1735196847782,"duration":0}},{"uid":"df0221dfd6ad2e64","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196800214,"stop":1735196800214,"duration":0}},{"uid":"58a9221d21bddda0","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196746443,"stop":1735196746443,"duration":0}},{"uid":"9fdfecc89c470998","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196745029,"stop":1735196745029,"duration":0}},{"uid":"8af4ea1d5d70cf91","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196713233,"stop":1735196713233,"duration":0}},{"uid":"b7b33919743d76ce","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196667324,"stop":1735196667324,"duration":0}},{"uid":"e387be4b70a0a44d","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196606602,"stop":1735196606602,"duration":0}},{"uid":"398046d46e25cc36","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196577501,"stop":1735196577501,"duration":0}},{"uid":"3b055b9b74f73ad3","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196558919,"stop":1735196558919,"duration":0}},{"uid":"602d173d67ef044b","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196535199,"stop":1735196535199,"duration":0}},{"uid":"8e7d52fd3438277","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196462568,"stop":1735196462568,"duration":0}},{"uid":"b5e57fba2b208a96","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196438141,"stop":1735196438141,"duration":0}},{"uid":"1b1952b6f262e757","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196397830,"stop":1735196397830,"duration":0}},{"uid":"b707318d81cea95e","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196321975,"stop":1735196321975,"duration":0}},{"uid":"4f7850b54f4da6ed","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196259701,"stop":1735196259701,"duration":0}},{"uid":"65bafa0d9849da3e","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196256140,"stop":1735196256140,"duration":0}},{"uid":"aca36025417881a3","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196253079,"stop":1735196253079,"duration":0}},{"uid":"611e7d31d46f2fff","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196132410,"stop":1735196132410,"duration":0}},{"uid":"91547e4944b5e250","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196123742,"stop":1735196123742,"duration":0}},{"uid":"90e1aaa0fbf71864","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196123574,"stop":1735196123574,"duration":0}},{"uid":"cd89c9bc40ab75d3","status":"broken","statusDetails":"playwright._impl._errors.Error: Page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL\nCall log:\nnavigating to \"None/register/\", waiting until \"load\"","time":{"start":1735196120918,"stop":1735196120918,"duration":0}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["smoke"]},"source":"363a0ad5985fb3bb.json","parameterValues":["'menu_link_qr_create'","'2048x2048'","''"]}